runners/flink/src/main/java/org/apache/beam/runners/flink/LyftFlinkStreamingPortableTranslations.java

```diff
@@ -357,6 +377,65 @@ private void translateKinesisInput(
             .addSource(source, FlinkLyftKinesisConsumer.class.getSimpleName()));
   }
 
+  private void translateS3AndKinesisInputs(
+      String id,
+      RunnerApi.Pipeline pipeline,
+      FlinkStreamingPortablePipelineTranslator.StreamingTranslationContext context) {
+    RunnerApi.PTransform pTransform = pipeline.getComponents().getTransformsOrThrow(id);
+
+    ObjectMapper mapper = new ObjectMapper();
+    String programConfigPath;
+    JSONObject jsonContent;
+
+    try {
+      JsonNode params = mapper.readTree(pTransform.getSpec().getPayload().toByteArray());
+      Preconditions.checkNotNull(
+          programConfigPath = params.path(
+              "program_config_path").textValue(),
```

Comment:
What would that config path be in a Beam deployment?

Reply:
I would assume it would be similar to what we have in the dryft sdk at the moment where we provide a json file. Example: https://github.com/lyft/realtimeintegrity/tree/master/src/main/resources

Reply:
I prefer we don't integrate with dryft sdk code like we have done for the realtime* jobs. My reasons

1. In realtime* jobs, the config file is a large json containing information about Kinesis streams, S3 paths, events, feature names and sql.   
I don't see all of this needed now.   It's preferable if we provide a simpler API to register the kinesis streams, S3 paths and events like [lyft_multi_source_inputs.py](https://docs.google.com/document/d/1CT7dNUElLMl7S7FPTOljaS_4ORXMotqVNepuou2jyXo/edit#heading=h.u4x7hggancgo)

Reply:
Yeah, @tweise suggested the same thing. The problem is currently the kinesis properties are kinda hardcoded in Dryft SDK. Ill look at how to decouple them.